lib/main/STM32H7/MiddleWares/ST/STM32_USB_Device_Library/Class/MSC/Src/usbd_msc_bot.c

```diff
@@ -100,15 +100,15 @@ static void MSC_BOT_Abort(USBD_HandleTypeDef  *pdev);
 */
 void MSC_BOT_Init(USBD_HandleTypeDef  *pdev)
 {
-  USBD_MSC_BOT_HandleTypeDef  *hmsc = (USBD_MSC_BOT_HandleTypeDef *)pdev->pClassData;
+  USBD_MSC_BOT_HandleTypeDef  *hmsc = (USBD_MSC_BOT_HandleTypeDef *)pdev->pMSC_ClassData;
```

Comment:
This does not seem to be about UNUSED-ness.

Reply:
Good spotting! It's a part of CDC mod.

Reply:
Probably worthwhile to squash these two commits to avoid it being overlooked in the next library upgrade.